### PR TITLE
Imagery Fixes for release/2.0.4

### PR DESCRIPTION
**Pull request by michaelrogers**
Closes 
#5274 
### Describe your changes:

* Added a height fallback to catch certain cases where the height for the focusedImageElement is not specified
* Removed duplicated requestHistory request in mounted
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Commit messages meet standards?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [x] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/imagery/mixins/imageryData.js [hunk 1]
```diff
@@ -46,7 +46,6 @@ export default {
 
         // kickoff
         this.subscribe();
-        this.requestHistory();
```
Comment (akhenry) [inline src/plugins/imagery/mixins/imageryData.js:49]: This is used by two components - `ImageryView.vue` and https://github.com/nasa/openmct/blob/master/src/plugins/imagery/components/ImageryTimeView.vue We need to make sure we are not introducing a regression in the time view here.

**Review by akhenry — CHANGES_REQUESTED** [review]

File: src/plugins/imagery/mixins/imageryData.js (hunk 1 above)
Comment (shefalijoshi) [reply src/plugins/imagery/mixins/imageryData.js:49]: Oh yeah need to test both views for realtime and historical data.

Comment (michaelrogers) [reply src/plugins/imagery/mixins/imageryData.js:49]: Agreed, this change was actually already merged into master and isn't present in the release/2.0.4 branch. I had paired with Nikhil to troubleshoot the clear data imagery bug.

https://github.com/nasa/openmct/pull/5223/files

Comment (michaelrogers) [reply src/plugins/imagery/mixins/imageryData.js:49]: Agreed, this change was actually already merged into master and isn't present in the release/2.0.4 branch. I had paired with Nikhil to troubleshoot the clear data imagery bug.

https://github.com/nasa/openmct/pull/5223/files

**Review by akhenry — APPROVED** [review]

**Pull request merged**